README.md

```diff
@@ -61,31 +60,38 @@ These are the available configuration options:
           "VolumeDown": true
         }
       },
-      "macros": {
-        "Play Xbox 360": [
-          [ "gpio", "TV", 1],
-          [ "gpio", "Receiver", 1],
-          [ "gpio", "Xbox", 1],
-          [ "SonyTV", "Power" ],
-          [ "delay", 500 ],
-          [ "SonyTV", "Xbox360" ],
-          [ "Yamaha", "Power" ],
-          [ "delay", 250 ],
-          [ "Yamaha", "Xbox360" ],
-          [ "Xbox360", "Power" ]
-        ],
-        "Listen to Music": [
-          [ "gpio", "Receiver", 1],
-          [ "Yamaha", "Power" ],
-          [ "delay", 500 ],
-          [ "Yamaha", "AirPlay" ]
-        ],
-        "all off": [
-          [ "gpio", "TV", 0],
-          [ "gpio", "Receiver", 0],
-          [ "gpio", "Xbox", 0]
-        ],
-      },
+      "macros": [
+        {
+          "name": "Xbox360",
+          "sequence": [
+            [ "gpio", "TV", 1 ],
+            [ "gpio", "Receiver", 1 ],
+            [ "gpio", "Xbox", 1 ],
+            [ "SonyTV", "Power" ],
```

Comment:
I wonder if it makes sense to add `ir` as the first parameter here to specify which communication protocol to use. This might make further enhancements (`zwave`, `http`) easier.

Reply:
currently ```lirc``` is a optional first argument to serve this purpose. Possible are any permutations of: 
* remove all code for the optional first argument (less code, more text in config files, less implicit config)
* rename the optional argument to ```ir```